### DISCUSSION TITLE
Only good vibes - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#20] [General]
# Only good vibes - Alan Pope's blog

Just a few thoughts about the origin of Linux Matters podcast.
Prior art Over the thirteen years of the Ubuntu Podcast the presenter lineup, format, duration and frequency changed here and there. In the early days, we would record a segment, have a cup of tea, and then record another one. It was a long and laborious process that took up most of a Sunday afternoon. After a little while we tweaked things and settled into our stride.

https://popey.com/blog/2023/08/only-good-vibes/

**Comment by CodingHerbivore** [comment]
Alan, I was very disappointed when the Ubuntu Podcast ended. I loved the simple (I use that word positively) format of three dudes sitting around having a calm, upbeat chat about Linux for a reasonable amount of time. I don't need to know the latest news about some drama in some project I'll never use, nor do I need two hours of whining about Windows or whatever-distro-i-don't-like. I'm absolutely thrilled that you are back at it with Linux Matters. It scratches the itch no other Linux podcast does, although I do miss the old-timey music.

**Reply by popey** [reply]
Thanks for the kind words

**Comment by m6wiq** [comment]
I agree with @CodingHerbivore , we have really missed a podcast with a calm upbeat chat regarding linux. I was really sad when the Ubuntu podcast ended but understood why the decision was made. I am really glad you have started a new podcast with a better system that works for you!
